aten/src/ATen/Version.cpp

```diff
@@ -205,7 +205,7 @@ std::string show_config() {
 
   // TODO: do HIP
   // TODO: do XLA
-  // TODO: do MLC
+  // TODO: do MPS
```

Comment:
Do we want to add that here?
We already have the info about cross compilation above. This might be useful to have to debug packaging issues in the future.

Reply:
Okay will remove this for now.

Reply:
I didn't meant that we should remove the TODO. But that we should do it sooner rather than later.

Reply:
Ah, okay.